go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -268,6 +263,39 @@ func (c *Controller) syncServices(key string) error {
 			"Error listing Endpoint Slices for Service %s/%s: %v", namespace, name, err)
 		return err
 	}
+
+	// If idling enabled and there are no endpoints, we need to move the VIP from the main loadbalancer
+	// ,that has the reject option for backends without endpoints, to the idling loadbalancer, that
+	// generates an event to kick of the unidling process in the openshift controller manager.
+	// if not, we delete the vips from the idling lb and move them to their right place.
+
+	vips := collectServiceVIPs(service)
+	if svcNeedsIdling(service.Annotations) && !util.HasValidEndpoint(service, endpointSlices) {
+		err = c.addServiceToIdlingBalancer(vips, service)
+		if err != nil {
+			c.eventRecorder.Eventf(service, v1.EventTypeWarning, "FailedToAddToIdlingBalancer",
+				"Error trying to add to Idling LoadBalancer for Service %s/%s: %v", name, namespace, err)
+			return err
+		}
+		vipsTracked = vipsTracked.Delete(vips.List()...)
+		err = deleteVIPsFromOVNBalancers(vips, name, namespace)
+		if err != nil {
+			c.eventRecorder.Eventf(service, v1.EventTypeWarning, "FailedToDeleteOVNLoadBalancer",
+				"Error trying to delete the OVN LoadBalancer for Service %s/%s: %v", name, namespace, err)
```

Comment:
should this be more descriptive to give some more context like:
"Error trying to delete OVN LoadBalancer while setting up Idling for Service..."